soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1587,6 +1602,280 @@ std::string Randomizer::GetGanonHintText() const {
     return ganonHintText;
 }
 
+ScrubIdentity Randomizer::IdentifyScrub(s32 sceneNum, s32 actorParams, s32 respawnData) {
+    struct ScrubIdentity scrubIdentity;
+
+    scrubIdentity.scrubId = -1;
+    scrubIdentity.randomizerCheck = RC_UNKNOWN_CHECK;
+    scrubIdentity.getItemId = GI_NONE;
+    scrubIdentity.itemPrice = -1;
+    scrubIdentity.isShuffled = GetRandoSettingValue(RSK_SHUFFLE_SCRUBS) > 0;
+
+    switch (actorParams) {
+        case 0x00:
```

Comment:
any chance we could make an enum with these values so it's not magic numbery? (i know we have other switch cases full of magic numbers in randomizer.cpp but it'd be nice to move away from that

Reply:
actually, it's not clear to me what the purpose of this part is. is this for having items for after we've gotten the "mysterious" item?

Reply:
coming back to this after seeing how it's used, seems it's an ogID thing